### ISSUE TITLE
bigip_ucs requires a refactor to use recent coding conventions

### ISSUE THREAD

**Issue by caphrim007** [F5Networks/f5-ansible#266]
One of the customer BIG-IP UCS module and is running into the following error:
 
‘msg: global name ‘test_icontrol’ is not defined’
 
They are able to use a workaround using the BIG-IP command module.

**Comment by jat190** [comment]
Can you submit all the relevant information and troubleshooting steps mentioned below?

##### COMPONENT NAME


##### ANSIBLE VERSION

```

```

##### PYTHON VERSION

```

```

##### BIGIP VERSION

```

```

##### CONFIGURATION


##### OS / ENVIRONMENT


##### SUMMARY


##### STEPS TO REPRODUCE



```

```



##### EXPECTED RESULTS


##### ACTUAL RESULTS



```

```

**Comment by caphrim007** [comment]
##### COMPONENT NAME

bigip_ucs.py

##### ANSIBLE VERSION

```
ansible 2.4.0
  config file = 
  configured module search path = [u'/Users/trupp/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/trupp/src/envs/ansible-2.0/lib/python2.7/site-packages/ansible
  executable location = /Users/trupp/src/envs/ansible-2.0/bin/ansible
  python version = 2.7.10 (default, Feb  7 2017, 00:08:15) [GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.34)]
```

##### PYTHON VERSION

```
Python 2.7.10
```

##### BIGIP VERSION

```
All versions that the Ansible modules support (12.x to present)
```

##### CONFIGURATION


##### OS / ENVIRONMENT


##### SUMMARY

The bigip_ucs.py file is reporting an error,

`msg: global name ‘test_icontrol’ is not defined`

##### STEPS TO REPRODUCE

Any usage of the module



##### EXPECTED RESULTS

The module should succeed instead of failing

##### ACTUAL RESULTS

The following error is raised


```
msg: global name ‘test_icontrol’ is not defined
```

**Comment by caphrim007** [comment]
@jat190 see above. There is no workaround right now. I am re-factoring the ucs module so that it will work.

**Comment by caphrim007** [comment]
@jat190 I believe this is now fixed, but will require some fixes merged into upstream Ansible before it is easily testable by others. I will update this PR when those are merged.

when they are, you will want to try the development version of Ansible.

**Comment by caphrim007** [comment]
@jat190 the required fix is now merged upstream so that it can be tested on a 2.4 devel version of ansible.
